torch/_functorch/aot_autograd.py

```diff
@@ -3913,11 +3951,25 @@ def backward(ctx, *flat_args):
 Expected grad_output types: {str(CompiledFunction.metadata.output_types)}
 Got grad_output types: {str(grad_output_types)}"""
 
-            del contiguous_args
-
             # TODO: figure out how to refactor the backward properly so I can use aot_dispatch_subclass_wrapper() here.
+            tangents_start_idx = len(all_args) - num_flat_bw_args_with_grads - len(rng_args)
+            tangents_end_idx = len(all_args) - len(rng_args)
```

Comment:
I would also move these index calculations to right where `all_args` is defined too, for clarity (so it's super clear why you computed it this way).